src/components/card/desktop/Card.tsx

```diff
@@ -47,7 +47,6 @@ const cardSizeStyles = {
 type CardProps = {
   className?: string
   id: ID
-  userId: ID
```

Comment:
If we drop `userId` then we have to be sure that the `id` prop passed into card is always the userId because this passed into UserBadges and it expects an `userId`

Reply:
gotchu gotchu